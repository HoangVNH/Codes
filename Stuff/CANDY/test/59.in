10
100
DDDXXDDDVDVDDVVDDVVVXVDVXVXXDXDXVXVXDDXDDVXDDVDXXVVVVXXXXVVXVXVVXDVDDVDVDXXVXVDDDDDVDVDDXDVXDXDXVXVX
63
XVVVDDXVDVDDXDVVXDXDDDVVDXDDXVXXVVVDVVDXDDDVVVVXDVDDVDXVVXXVVVV
62
XXXDDVXVXVVVVXVXXDDVDDXDVDDVDXDXDDDVXVXVDXDXXXXVXDVVDDVDXDVVDX
54
DVXDDXDXDVVVDVVXDDDXDDDVVDXXXXDXXXXDVXDXVXXDVDVVXXVXDX
76
VVDDXXDDXDDXVXDXXDVDXVVDDXDXDXXXDXXDXVXDDDVXXVDXDVVDVVXVVDXXDDVXVVVXDDDXXXVV
80
XDXXXVXDXXDVXXXVXVDDXDXVXDVXXDDDVVVXVDDXVDVVVXDDVDVVXVVVDDDXXXVVVDVVVDXDVDDXVXXV
84
VVVVVXDXDDXVDXDDVDVDXDXXDXDXXXXDDDVXVVDDVXVVDVDVXXVVDVVDDVXDDDDVDDVDVVVDDDVDVVVVVDVX
51
XXXDDVVXDXVDDDVXDXDVVDXVDDDVDVVVVVDXXDVXDXDXVVXDVXV
55
VDDXDVXXXDDXXXDVVXDVVDDXDVXXXXDXDDDVVDDVVXXDXXDVXXXVVXX
73
DVXXVDXDVXDXDVVDDXVDVXXDDXDDXXDDVXDDVXXXXVXXXVDDDXXXXVVXXXXVXVDDVDDXVDDDD
