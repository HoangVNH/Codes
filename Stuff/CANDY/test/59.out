99
62
61
53
75
79
83
50
54
71
